templates/careers/application/index.html

```diff
@@ -220,15 +222,11 @@ <h1>Thanks for applying {{ application["candidate"]["first_name"] }} {{ applicat
         </p>
       </div>
     </section>
-  {% else %}
-    <section class="p-strip">
+    {% elif application["status"] != "active" and application["rejection_reason"]["type"]["id"] == 2 %}
```

Comment:
I _think_ this means that the application is not active and it's because the user has withdrawn the application, rather than been rejected - it's tricky to know for sure, it might be better to move this check to the backend, and expose it to the template via some more readable variable, i.e. `withdrawn` 
```suggestion
    {% elif withdrawn %}
```

Similar to the use of `withdrawal_requested` above on  line192

Reply:
Agree. This currently seems to be duplicated, same code is in the Python code